mixer/cmd/root.go

```diff
@@ -75,8 +75,17 @@ var RootCmd = &cobra.Command{
 			fail(err)
 		}
 
+		networkCheck := true
+		noNetworkCmds := []string{"list", "edit", "validate", "convert", "set", "repo", "add-rpms"}
```

Comment:
Hmm, I don't really like this approach since it ignores commands subcategories. Something like "set" is so generic that it can easily end up in multiple categories.

Other than printing the warnings, this block doesn't do anything else. Wouldn't it be better to just move it to the commands where the warning is relevant?

Reply:
I'm open to updating this, this was just an initial list I came up before asking anyone for RFC's!

That being said "set" likely is never going to set something over the network, so it's not a bad catch-all.